### ISSUE TITLE
RHEL 9 dependencies could be wrong

### ISSUE THREAD

**Issue by johrstrom** [OSC/ood-documentation#814]
There seems to be an issue with how RHEL/9 is documented. See this discourse topic.

https://discourse.openondemand.org/t/ood3-installation-on-rhel9/2692/2

I'm finding that our CI that builds our packages runs ruby 3.0 which is fine, but nodejs version is actually 16. At least in `EL 9.1`. 

From the discourse topic, it could be the case that these packages have been updated in other minor versions of EL 9 and/or streams changed.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1204613972677264) by [Unito](https://www.unito.io)

**Comment by johrstrom** [comment]
I think I've confirmed this. There's no need to enable modules for RHEL/9 - the base packages of ruby 3.0 and nodejs 16 are the packages that you require.

**Comment by treydock** [comment]
What happens here is RHEL9 has fewer modules than RHEL8.  When RHEL8 came out they started with multiple versions of Ruby, NodeJS, etc.  With RHEL9 they dialed that back a bit so first few releases of RHEL9 only had a single module for most things and those default modules in OnDemand's case is Ruby 3.1 and NodeJS 18.  If RHEL9 comes out with like ruby:3.2 module and we need that one, then there will be extra steps.

Example:

```
$ docker run --rm -it --platform linux/amd64 rockylinux:9 /bin/bash
[root@c6762c66b85d /]# dnf install -y https://yum.osc.edu/ondemand/3.0/ondemand-release-web-3.0-1.noarch.rpm epel-release
[root@c6762c66b85d /]# dnf install -y ondemand
```

Normally if DNF was enabling a non-default module, it would output that but the above steps don't output anything about modules as we are using all the defaults.  

Looks like default Ruby is 3.1 and nodejs is 18.  I Think the fact we are seeing 3.1 is a bug in how downstream RHEL clones handle modules.  I'm fairly certain RHEL 9 supports both Ruby 3.0 and 3.1 but I think the clones like Alma and Rocky only support 3.1.  https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle#rhel9_full_life_application_streams

**Comment by johrstrom** [comment]
This was fixed in #904.
